:100755 100755 18ce48ca7 000000000 M	cmake/install/post/generate-adios2-config.sh.in

diff --git a/cmake/install/post/generate-adios2-config.sh.in b/cmake/install/post/generate-adios2-config.sh.in
index 18ce48ca7..4603420ff 100755
--- a/cmake/install/post/generate-adios2-config.sh.in
+++ b/cmake/install/post/generate-adios2-config.sh.in
@@ -86,7 +86,10 @@ then
   export FC="@MPI_Fortran_COMPILER@"
 fi
 
-export adios2_DIR="${PREFIX}/@CMAKE_INSTALL_CMAKEDIR@"
+if [ -z "$adios2_DIR" ]
+then
+  export adios2_DIR="${PREFIX}/@CMAKE_INSTALL_CMAKEDIR@"
+fi
 
 # Sometimes package managers build in root jails
 if [ ! -d "${adios2_DIR}" ]
